app/page/template/content/conversation/input-bar.htm

```diff
@@ -20,16 +20,15 @@
                              enter: onInputEnter,
                              hasFocus: hasFocus,
                              textInput: input,
-                             resize: input,
-                             resizeCallback: scrollMessageList,
+                             resize: {triggerObservable: input(), callback: scrollMessageList, syncElement: '.shadow-input'},
```

Comment:
Let's rename `triggerObservable` `triggerValue`, it's not an observable anymore because we resolve it